### ISSUE TITLE
Make it possible to invert a filter selection

### ISSUE THREAD

**Issue by tmadlener** [key4hep/eede#72]
Once #70 is merged and filters are working again, I think it would be nice to have a way to invert a filter selection, such that only the objects for which the filter is false are shown.

E.g. at the moment I can filter on the simulation status to only get the particles that were created during simulation. However, quite frequently, I would also like to just have all the particles that were not created during simulation. I think as a first step it would be enough to simply make it possible to negate the full filter expression that users come up with.
